models/tranquil/tranquil_txs.sql

```diff
@@ -0,0 +1,131 @@
+{{
+    config(
+        materialized='table',
+        unique_key='log_id',
+        tags=['core', 'defi', 'amm', 'lending']
+        )
+}}
+
+with
+
+tranquil_market_labels as (
+    select
+        *
+    from {{ ref('tranquil_market_labels') }}
+),
+
+-- Tranquil contracts all have the same "Creator" or admin address
+tranquil_contracts as (
+    select distinct
+        evm_contract_address
+    from logs 
```

Comment:
{{ ref('logs') }}

Reply:
also is this materialised as a table, as in it will run a full refresh every single time or it should be an incremental build?